consensus/src/consensus_provider.rs

```diff
@@ -80,7 +80,23 @@ pub fn start_consensus(
     let (network_task, network_receiver) = NetworkTask::new(network_events, self_receiver);
 
     runtime.spawn(network_task.start());
-    runtime.spawn(epoch_mgr.start(timeout_receiver, network_receiver));
+    let metrics_monitor = tokio_metrics::TaskMonitor::new();
+    let epoch_manger_instrumneted =
+        metrics_monitor.instrument(epoch_mgr.start(timeout_receiver, network_receiver));
+
+    let metrics_frequency = std::time::Duration::from_secs(1);
```

Comment:
This is too frequent and will spam the logs, lets make it 30 s or so?

Reply:
Ack